features/data_converter/json_protobuf/feature.ts

```diff
@@ -0,0 +1,67 @@
+import { Feature } from '@temporalio/harness';
+import * as proto from '@temporalio/proto';
+import { patchProtobufRoot } from '@temporalio/proto/lib/patch-protobuf-root';
+import { fromProto3JSON } from 'proto3-json-serializer';
+import { decode } from '@temporalio/common/lib/encoding';
+import * as assert from 'assert';
+
+// Cast to `any` because the generated proto module types are missing the `lookupType` method
+const patched = patchProtobufRoot(proto) as any;
+const dataBlobType = patched.lookupType('temporal.api.common.v1.DataBlob');
+
+// Inject Buffer and Uint8Array into isolate to workaround SDK bug
+// TODO(antlai-temporal) Remove workaround when SDK bug is fixed
+const g = globalThis as any;
+g.Uint8Array = g.constructor.constructor('return globalThis.Uint8Array')();
```

Comment:
Don't think you'll need to inject `Uint8Array` here, just `Buffer` AFAIR.

Reply:
You are right, it just needs Buffer